src/rrc/io/unix/UnixFile.cpp

```diff
@@ -0,0 +1,39 @@
+/**
+ *  @autor nikitas
+ *  @date 10/12/16
+ */
+
+#include <fcntl.h>
+#include <unistd.h>
+
+#include "rrc/io/unix/UnixFile.h"
+
+rrc::unix::UnixFile::UnixFile() : mFileDescriptor(-1) {
+
+}
+
+rrc::unix::UnixFile::UnixFile(const std::string &path, rrc::Mode mode) {
+    this->open(path, mode);
+}
+
+void rrc::unix::UnixFile::open(const std::string &path, rrc::Mode mode) {
```

Comment:
Go to guidelines. As you can see there are a lot of places with the same formatting troubles. So just fix it everywhere)